Vb/databricks admin

vb-ssc opened the pull request:
<!-- Provide a general summary of your changes in the Title above -->

## Description
Create button in workspace SUDO page for system admins to add themselves to databricks workspace admin group via REST api

## How Has This Been Tested?
- tested manually 

## Types of changes
<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Unit test (new or update to tests)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (change that would cause documentation to be updated)
- [ ] Configuration change (change that would affect the configuration of the application)

<!-- If you made any visual changes, provide screenshots of before/after, it its moving parts, please provide high quality gif, wemb or mp4 -->

## Checklist:
<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!-- If you're unsure about any of these, don't hesitate to ask! -->
- [x] PR is submitted to the correct branch `develop`.
- [x] Code follows the code style of this project.
- [ ] Changes are covered by Specflow tests and all new or existing tests are passing.
- [x] Any new or updated translatable strings have been added to the localization files.
- [ ] Necessary and relevant documentation has been created or updated.
- [ ] There are either no changes to the application's configuration or the necessary changes in the infrastructure repository are included in a separate PR.

<!-- Link to the "infrastructure" repository PR here -->

vb-ssc committed:
2095 Updated SUDO page to include new action button

vb-ssc committed:
fixes

vb-ssc committed:
usinf DataBricks scope to authenticate

vb-ssc committed:
using elevated permissions to generate token

vb-ssc committed:
set user active

vb-ssc committed:
added unit tests

vb-ssc committed:
added localization and unit tests

yjmrobert commented:
Some failing unit tests, I will merge develop in and see if that resolves it. Be sure to double check they are all passing!

yjmrobert committed:
Merge branch 'refs/heads/develop' into vb/databricks-admin

yjmrobert committed:
🧪 updated test

yjmrobert committed:
🔧 Merge and update AzureDevOpsClient method names

This commit modifies method names in AzureDevOpsClient to standardize asynchronicity conventions. It also refactors the pull request approval logic in RepositoryService to automatically complete PRs rather than auto-approving them. The PR completion process now uses a dedicated service user rather than the previous auto-approve user. An adjustment was made to the appsettings files, and updates were applied to connect PostgreSQL in workspace databases using R and Python.

yjmrobert committed:
📜 updated unit test with new method name
